training/distributed_training/pytorch/data_parallel/mnist/code/train_pytorch_smdataparallel_mnist.py

```diff
@@ -38,26 +38,28 @@
 # Reference PR: https://github.com/pytorch/vision/pull/3559
 TORCHVISION_VERSION = "0.9.1"
 if Version(torchvision.__version__) < Version(TORCHVISION_VERSION):
+    # Set path to data source and include checksum key to make sure data isn't corrupted
     datasets.MNIST.resources = [
         (
-            "https://dlinfra-mnist-dataset.s3-us-west-2.amazonaws.com/mnist/train-images-idx3-ubyte.gz",
+            "https://sagemaker-sample-files.s3.amazonaws.com/datasets/image/MNIST/train-images-idx3-ubyte.gz",
             "f68b3c2dcbeaaa9fbdd348bbdeb94873",
```

Comment:
Did you check to make sure the checksums are the same between the two data versions? It should be the same data, but I'm not positive there isn't some small difference.

Reply:
Yes, I looked into it and there were using the exact same files as us. Also, I tested everything, so if the checksum didn't work, I wouldn't be able to run the notebooks end-to-end